setup.py

```diff
@@ -264,6 +264,7 @@
     'obspy.plugin.waveform.AH': [
         'isFormat = obspy.io.ah.core:_is_ah',
         'readFormat = obspy.io.ah.core:_read_ah',
+        'writeFormat = obspy.io.ah.core:_write_ah1'
```

Comment:
there is another format: AH2, which is apparently never used. The AH files in AH1 format I saw so far all used the AH or ahx suffix